model/model.py

```diff
@@ -184,45 +189,46 @@ def validate_batch(self):
 
     # TODO funktionen gör alldeles för mycket,
     # dela upp utskrift, beräkning och träning
-    def train(self):
+    def train(self, use_pretrained_net=False):
         """ Trains the model on the dataset """
         print("Starting training...")
 
-        if self.use_pretrained:
+        if self.use_pretrained and \
```

Comment:
We only want to initialise the embedding matrix once but we run train twice therefore we want to run it only if both self.use_pretrained_net and use_pretrained_net is true i.e. we want to pre-train on subreddits and this call to train will pre-train or if both of them are false i.e. we don't want to pre-train and this call don't do pre-train. 
That's just bad naming, the class variable is whether the model should use pre-training on the network and the parameter is whether this call to the method should pre-train the network